app/liquid_tags/open_graph_tag.rb

```diff
@@ -0,0 +1,21 @@
+class OpenGraphTag < LiquidTagBase
+  # OpenGraphTag is NOT registered in the Registry;
```

Comment:
When building the API documentation via Yard, the comments are associated with the line of code that follows the comments.

I believe this comment makes more sense as a top-level comment (e.g. move to above line 1)